src/go/cmd/token_builders.go

```diff
@@ -0,0 +1,264 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package cmd
+
+import (
+	"strings"
+)
+
+func addTab() string {
+	return "     "
```

Comment:
Should this return "\t" instead?

Reply:
Done and removed addTab() since it does seem unnecessary